File: app/src/main/java/de/mchllngr/quickero/util/CustomAnimatorListener.java
package de.mchllngr.quickero.util;

import android.animation.Animator;

public interface CustomAnimatorListener extends Animator.AnimatorListener {

    @Override
    default void onAnimationStart(Animator animation) { /* empty */ }

    @Override
    default void onAnimationEnd(Animator animation) { /* empty */ }

    @Override
    default void onAnimationCancel(Animator animation) { /* empty */ }

    @Override
    default void onAnimationRepeat(Animator animation) { /* empty */ }
}

File: app/src/main/java/de/mchllngr/quickero/util/CustomAnimatorListener.kt
package de.mchllngr.quickero.util

import android.animation.Animator

interface CustomAnimatorListener : Animator.AnimatorListener {

    @JvmDefault // TODO remove after #7
    override fun onAnimationStart(animation: Animator) = Unit

    @JvmDefault // TODO remove after #7
    override fun onAnimationEnd(animation: Animator) = Unit

    @JvmDefault // TODO remove after #7
    override fun onAnimationCancel(animation: Animator) = Unit

    @JvmDefault // TODO remove after #7
    override fun onAnimationRepeat(animation: Animator) = Unit
}
